.dockerignore

```diff
@@ -2,3 +2,7 @@
 pkg
```

Comment:
this needs to be removed as most of the sysl code is in there. It wont build without this directory.